### ISSUE TITLE
Support inserting accessible images

### ISSUE THREAD

**Issue by nonprofittechy** [elapouya/python-docx-template#635]
## Is your feature request related to a problem? Please describe.
When I dynamically add an image to a DOCX file with python-docx-template, as a screen reader user, I am unable to assign or listen to alternative text for that image.

## Describe the solution you'd like
Modify the InlineImage class so that it has an `alt_text` and `title` property. Use this to set the "descr" and "title" attributes of the newly inserted XML element.

## Describe alternatives you've considered
This feature is available in the python-docx layer, but not when using python-docx-template. Only true alternative therefore is editing it after processing, either manually or through using the python-docx library.

## Additional context
This is an important accessibility feature that impacts WCAG compatibility, as well as PDF/UA if the DOCX is later converted to PDF.
